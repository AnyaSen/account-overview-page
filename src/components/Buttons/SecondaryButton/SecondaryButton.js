import React from "react";
import Styles from "./SecondaryButton.module.scss";


export default function SecondaryButton({ btnText }) {
  return <button className={Styles.SecondaryButton}>{btnText}</button>;
}
