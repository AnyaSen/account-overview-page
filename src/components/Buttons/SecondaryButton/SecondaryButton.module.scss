@import '../../../assets/scss/mixins';

.SecondaryButton {
	@include button($color-light-grey, $color-light-grey, $color-grey);
}
