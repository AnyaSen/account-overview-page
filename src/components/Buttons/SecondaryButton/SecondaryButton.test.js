import React from "react";
import { render, cleanup } from "@testing-library/react";

import SecondaryButton from "./SecondaryButton";

describe("<SecondaryButton />", () => {
  afterAll(cleanup);

  test("Should have correct text and be disabled.", () => {
    const text = "Show All";
    const isDisabled = true;

    const { getByText, container } = render(
      <SecondaryButton btnText={text} isDisabled={isDisabled} />
    );

    const button = getByText(text);

    expect(button).toBeTruthy();
    expect(button).toHaveAttribute("disabled");

    expect(container.firstChild).toMatchSnapshot();
  });
});
