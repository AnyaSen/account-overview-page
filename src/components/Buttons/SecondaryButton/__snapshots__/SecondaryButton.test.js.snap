// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SecondaryButton /> Should have correct text and be disabled. 1`] = `
<button
  class="SecondaryButton"
  disabled=""
>
  Show All
</button>
`;
